<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>261</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="side_menu_widget" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget{
	background-color: #38686A;
	font-family: 'Segoe UI';
}

QPushButton{
	color: #FFFFFF;
	background-color: #A3B4A2;
	font-size: 23px;
	border: none;
	border-top-left-radius: 15px;
	border-bottom-left-radius: 15px;
	height: 50px;
	text-align:left;
	padding-left: 10px;
}

QPushButton:checked{
  	background-color:#FFFFFF; 
 	color: #38686A; 
}
</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>251</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn_tela_inicio">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Início</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/icons/icons/house-chimney.png</normaloff>
             <normalon>:/icons/icons/house-chimney.svg</normalon>:/icons/icons/house-chimney.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_tela_vendas">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Vendas</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/icons/icons/point-of-sale-bill.png</normaloff>
             <normalon>:/icons/icons/point-of-sale-bill.svg</normalon>:/icons/icons/point-of-sale-bill.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_tela_produtos">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Produtos</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/icons/icons/box-open.png</normaloff>
             <normalon>:/icons/icons/caixa-aberta.svg</normalon>:/icons/icons/box-open.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_tela_clientes">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Clientes</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/icons/icons/user-pen.png</normaloff>
             <normalon>:/icons/icons/user-pen.svg</normalon>:/icons/icons/user-pen.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>1031</width>
      <height>721</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <widget class="QWidget" name="page_inicio">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>471</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>40</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Início</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionmenu_item_2">
   <property name="text">
    <string>opt-1</string>
   </property>
  </action>
  <action name="actionopt_2">
   <property name="text">
    <string>opt-2</string>
   </property>
  </action>
  <action name="actionopt_3">
   <property name="text">
    <string>opt-3</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionopt_1">
   <property name="text">
    <string>opt-1</string>
   </property>
  </action>
  <action name="actionopt_4">
   <property name="text">
    <string>opt-2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
